### ISSUE TITLE
230V ready?

### ISSUE THREAD

**Issue by nikwest** [dragonlock2/OSRO#1]
Love the design. From what I can tell it should also work with 230V AC?

**Comment by dragonlock2** [comment]
I haven't taken a look at this design in a while, but on a quick analysis it should work. Worst case some of the resistors in the switching circuit need to be resized.

**Comment by nikwest** [comment]
ok, thank you. I'll give it a try and let you know, if it works. Any particular resistor you have in mind? I was looking at R12, but as far as I understood it, this is suppose to limit the current and the value is depending on the voltage drop of the triac. So here the AC voltage doesn't really matter.

**Comment by dragonlock2** [comment]
Actually the only thing I'd change would be to increase R8/R9 value to 22k or so since power dissipation would otherwise increase past the 1206 limit. Everything else should be fine at 230V AC. Hope it works :)

**Comment by nikwest** [comment]
Good point. Didn't think about that.

**Comment by nikwest** [comment]
I can't really get resistors with correct ratings. Especially the 10k R13 and R16 I can't get with 400V rating.  I'll need to split it up into 2 5k 200V 803 resistors. Also I think it's better to use 33k for R8/R9 and SFH6206-3 instead of SFH620A-2, which has a slightly better CTR rating.
Therefore I have to touch the PCB layout. The extraparts and esspressif footprint folders are empty in the repository. I guess you haven't committed them due to copyright restrictions?

**Comment by dragonlock2** [comment]
Good point on the resistor voltage rating, high voltage ones are harder to find. SFH6206-3 should be fine, parts I choose are what was cheapest on DigiKey at the time or what I had on hand so there's room for optimization.

`extraparts` and `espressif` are submodules, either do a `git clone --recursive` when cloning or a `git submodule init && git submodule update` after cloning to check those out.

**Comment by nikwest** [comment]
I made a couple of changes to your design and adapted it for 230V. The first build looks like it works nicely, except that my oven is not strong enough and can't follow the profile temperature so quickly. 
Question is, if you want a PR for this to incorporate in your repository or rather keep this version separate?
I also changed from USB-B to a USB-C connector, and also the wire connectors and for my second board I like to use the MAX31855 instead of MAX6675. They are pin compatible, but firmware needs to be adapted.

![IMG_0580](https://github.com/dragonlock2/OSRO/assets/42672/43854349-803f-46cd-bf48-59e4b299240d)

**Comment by dragonlock2** [comment]
That looks great! Glad that it's working. I'm a bit lazy to figure the best way to organize two board variants so let's keep your changes in a fork for now. If there's more community interest I'll take a closer look

**Comment by nikwest** [comment]
Yes, I agree, just wanted to check back with you. Then I'll keep the fork.
